content/concepts/peer-routing.md

```diff
@@ -3,6 +3,82 @@ title: Peer Routing
 weight: 5
 ---
 
-This article is coming soon!
+One of the many mechanisms that libp2p uses for peer routing is Kademlia DHT. We will dive into how it works in this article, this will help you further understand more advanced topics.
 
-Please [refer to this issue](https://github.com/libp2p/docs/issues/13) to track the progress and make suggestions.
+## Static DHT
+
+To grow intuition about the Kademlia let's assume for now that our DHT is static. No one joins DHT and no one leaves.
+
+### System Descritpion
+
+Each peer has their own peer ID. Kademlia provides an efficient lookup algorithm that locates successively “closer” nodes to any desired ID.
+
+In Kademlia peers are key-value pairs in the DHT. Conceptually peer ID stored as a key and [address](/concepts/addressing/) (or IP address) of a peer stored as a value.
+
+Kademlia treats peers as leaves in a binary tree. We store hashes of Peer IDs in leaves, which are represented as binary numbers. Notice, that we do not need to store the whole hash to specify the peer. We can store only prefix if prefix is unique.
+
+Leaves on the tree that are not peers represent [content](/concepts/content-routing/) which is stored by some close nodes.
+
+![Kademlia-system](Kademlia-system.png)
+
+Red node here has prefix `001`, go from the top to the bottom by edges. Let's say it' Bob. Alice here has prefix `11101` - she is blue node. Yellow leaves are other peers. For any given node, Kademlia divides the whole binary tree into a series of successively lower subtrees that don’t contain the given node. The highest subtree consists of the half of the binary tree not containing the node. The next subtree consists of the half of the remaining tree not containing the node, and so forth. With example it'll be more clear. You can see here for node `001` subtrees are circled and consist of all nodes with prefixes `1`, `01`, `000`.
+
+The Kademlia protocol ensures that every node knows of at least one node in each of its subtrees, if that subtree contains a node. This guarantees that any node can be located by it's peer ID.
+
+Let's see how Bob can locate Alice in this example. Bob knows Alices peerID, but he needs to get the value - Alices IP address. From subtree with prefix `1` Bob knows peer with prefix `100`. He sends a message to their address, where he asks for Alices address.
```

Comment:
```suggestion
Let's see how once can locate in an example. We will call the red node 'Bob', with the prefix  `001.` We will call the yellow node Alice, with the prefix `01`

Bob can locate Alice in this example.  Bob knows Alice's peerID, but he needs to get Alice's location address. From the subtree with the prefix `1` Bob knows a peer with prefix `101`. He sends a message to their address, where he asks for Alice's address.
```

Reply:
I'm not entirely sure what you mean by this. The words address, key, location address keep being used interchangeably. I recommend you review all of this paragraph to accurately describe what is being located. Is it a PeerID? IP Address?

Reply:
Yeah, i'm sorry for this mess.

Here DHT is a storage for key-value pairs. 
In our case key is Alice's peer ID. Bob knows it, and he needs to get the value.
Value here is Alice's IP address, so then Bob will be able to communicate directly with this IP address (and this is Alice)

I added "a brief reminder" paragraph in my current commit and I hope it'll make it clearer.